airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -40,17 +39,13 @@ class HttpStream(Stream, ABC):
     page_size: Optional[int] = None  # Use this variable to define page size for API http requests with pagination support
 
     # TODO: remove legacy HttpAuthenticator authenticator references
```

Comment:
```suggestion
```
This is probably the only one but might be worth a quick comment search for references!